client.go

```diff
@@ -1547,6 +1559,53 @@ func (c *ToznySDKV3) WriteFile(ctx context.Context, recordType string, plain map
 	return fileRecord, nil
 }
 
+type ReadFileOptions struct {
+	RecordID         string
+	DownloadFileName string
+}
+
+// ReadFile downloads and decrypts the file from the record
+func (c *ToznySDKV3) ReadFile(ctx context.Context, options ReadFileOptions) error {
+	secretUUID, err := uuid.Parse(options.RecordID)
+	if err != nil {
+		return err
+	}
+	fileResp, err := c.E3dbPDSClient.GetFileRecord(ctx, secretUUID)
+	if err != nil {
+		return err
+	}
+	fileURL := fileResp.Metadata.FileMeta.FileURL
+	downloadRequest := file.DownloadRequest{
+		URL:               fileURL,
+		EncryptedFileName: DefaultDownloadedFileName,
+	}
+	downloadResponse, err := file.DownloadFile(downloadRequest)
+	if err != nil || downloadResponse != "" {
+		return fmt.Errorf("ReadFile: Err: %+v  Resp: %+v", err, downloadResponse)
+	}
+	defer func() {
+		err := os.Remove(DefaultDownloadedFileName)
+		if err != nil {
+			fmt.Printf("ReadFile: Could not delete %s: %+v", DefaultDownloadedFileName, err)
+		}
+	}()
+	keyRequest := pdsClient.GetOrCreateAccessKeyRequest{
+		WriterID:   c.E3dbPDSClient.ClientID,
+		UserID:     c.E3dbPDSClient.ClientID,
+		ReaderID:   c.E3dbPDSClient.ClientID,
+		RecordType: fileResp.Metadata.Type,
+	}
+	ak, err := c.E3dbPDSClient.GetOrCreateAccessKey(ctx, keyRequest)
+	if err != nil {
+		return err
+	}
+	err = e3dbClients.DecryptFile(DefaultDownloadedFileName, options.DownloadFileName, ak)
+	if err != nil {
+		return err
+	}
+	return nil
```

Comment:
nit: some story comments to break up the code sections would be nice (download the file, get the access key, decrypt it, etc....)